googlemock/src/gmock-spec-builders.cc

```diff
@@ -487,8 +487,24 @@ bool UntypedFunctionMockerBase::VerifyAndClearExpectationsLocked()
       // takes care of it.
       untyped_expectation->MaybeDescribeExtraMatcherTo(&ss);
       untyped_expectation->DescribeCallCountTo(&ss);
-      Expect(false, untyped_expectation->file(),
-             untyped_expectation->line(), ss.str());
+      if (! registry_destructed_) {
+        Expect(false, untyped_expectation->file(),
+               untyped_expectation->line(), ss.str());
+      } else {
+        // Program nearing end: static registry already destructed.
+        // Other statics (such as obj pointed to by return of
+        // UnitTest::GetInstance(); which is called by Expect)
+        // probably also already destructed...
+        // -> Write out message immediately, instead of calling Expect.
+        std::cout << "   Warning: Mocks (typically mocks with static storage duration), "
```

Comment:
The phrasing is weird. Eliminate the parens. The parens are optional information.
If you take them out, you have a false statement.
Mocks can normally be verified on destruction.